Makefile

```diff
@@ -723,19 +511,93 @@ manual-pregen: opt.opt
 # The clean target
 clean:: partialclean
 
+# Prefixed compiler-libs
+
+compilerlibs/ocaml_common__compdynlink.cmo: \
+	compilerlibs/ocaml_common__compdynlink.mlbyte
+	$(CAMLC) $(COMPFLAGS) $(shell compilerlibs/Compflags $@) -c -impl $<
```

Comment:
@shindere will be sad to see a new `Compflags` script added to the build system...